tests/generate_proto.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+PROTOC=protoc
+
+MAIN_DIR=proto_dir/proto/main
+PROTO_PATH=proto_dir/proto/main/exonum
+SERVICE_DIR=proto_dir/proto/exonum_cryptocurrency_advanced_0_11_0
```

Comment:
Nit: These files ain't correspond to 0.11.0 anymore, so probably it will be a good idea to rename the module.